ingest/ingest.py

```diff
@@ -1,359 +0,0 @@
-import yaml
-import requests
-import json
-import os
-from os.path import expanduser
-from subprocess import call,check_output,Popen,PIPE
-import subprocess
-import json,pickle
-import shutil 
-import glob2
-import boto3
-from datetime import datetime
-
-
-with open("config.yml", 'r') as stream:
-    config = yaml.load(stream, Loader=yaml.FullLoader)
-
-orgs = config['orgs']
-ind_repos = config['individual_repos']
-cloned_project_path = expanduser("~") + '/cloned_projects'
-
-def get_org_repos():
-    org_repos = []
-    for org in orgs:
-        org_repos_response = requests.get('https://api.github.com/orgs/' + org + '/repos?access_token='+os.environ['github_access_token'])
-        org_repos = org_repos + json.loads(org_repos_response.text)
-
-    for ind in ind_repos:
-        response = requests.get('https://api.github.com/repos/' + ind + '?access_token='+os.environ['github_access_token'])
-        ind_repo = []
-        ind_repo.append(json.loads(response.text))
-        org_repos = org_repos + ind_repo
-    return org_repos
-
-def map_repo_attributes(org_repos):
-    repos = []
-    for org_repo in org_repos:
-
-        clone_dir = cloned_project_path+'/'+org_repo['owner']['login']+'/'+org_repo['name']
-        git_url = org_repo['clone_url']
-        adjusted_url = git_url.replace('https://','https://' + os.environ['github_user'] + ':' + os.environ['github_access_token']+'@')
```

Comment:
This should be `org_repo['owner']['login']` and not `os.environ['github_user'` I think.

Reply:
No, that's for the github api authentication, not for that particular repo.